semantic/src/site/globals/site.overrides

```diff
@@ -154,31 +154,42 @@ p strong {
   }
 }
 
-#application-area .ui.header .sub.header {
-  margin-top: 11px;
-  margin-bottom: 16px;
-  font-size: @fontLarge;
-  letter-spacing: @letterSpacingSmall;
+// Steps line (vertical box - layout)
+.line-vertical-box {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  min-height: 30px; // TODO: Need to be updated for mobile
+  align-items: center;
+  justify-content: space-between;
 }
 
-// Extra font and margin on Steps header on application home/create
-#application-area .steps-header {
-  font-size: @fontSmall;
-  font-weight: @regularWeight;
-  margin-top: 0;
-  margin-bottom: 5px;
+// Inner steps line (left side - icon & header)
+.line-vertical-box .centered-flex-box-row {
+  display: flex;
+  flex-grow: 1;
+  flex-direction: row;
+  align-items: center;
```

Comment:
I'm confused with your suggestion, sorry if I can't really follow. Code examples would be easier to follow up. What I was trying to achieve was a second flex component to have children layout inside `centered-flex-box-row`. I don't really see how to use the **same** class for both, and what are the benefits of listing a lot of generic classes on the semantic elements, this would make changing css a little more tricker than changing in a single place. : (

Reply:
Okay, I can see where I was kind of  mis-leading -- the "inner" element is actually a flex box in its own right (for the icon and name), so you do need those properties, but at least you've kept them separate now, so they're not "coupled" to one another unnecessarily.

I do think we could make more "generic" flex-box component classes that we can extend for more specificity, but what you've done now should be fine for the time being. I'll probably give them some further integration with my list styles when I merge this, and we should continue to refine as generic elements become more obvious.

You've got a lot of experience with an Object-oriented approach, right, so what I'm trying to do really is have a well-designed "inheritance tree", with some simple base classes and then some smaller, more specific classes for "extending" them. But it's probably best to wait until we have more to work with and do it incrementally.

Thanks for your patience.

Reply:
Hmmm I see, I think I got the idea a little bit better now. And yes, definitely easier to improve as we go. But I understand you would like to have a base start that isn't adding too many classes since this can get more difficult to revert.

Reply:
> But I understand you would like to have a base start that isn't adding too many classes since this can get more difficult to revert.

Yes, exactly.